package com.lec.ex2_throws;

public class Main {

	public static void main(String[] args) { //main함수까지 throw가 오면 --> JVM이 예외를 받음.
		new ThrowsEx();

	}

}
